### DISCUSSION TITLE
v0.63.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#3399] [Announcements]
### Enhancements
* The editor now supports file attachments in https://github.com/outline/outline/pull/3031
* We added search to publicly shared documents by @thenanyu in https://github.com/outline/outline/pull/3126
* It is now possible to star collections in https://github.com/outline/outline/pull/3327
* Outline will now work behind a [Pomerium](https://www.pomerium.com/) proxy in https://github.com/outline/outline/pull/3219
* You can now navigate to all the pages of settings through command bar by @iamsaumya in https://github.com/outline/outline/pull/3226
* Much improved settings layout in https://github.com/outline/outline/pull/3234
* Collection creation can now be disabled for members in https://github.com/outline/outline/pull/3270
* Added /date, /time, and /datetime patterns to insert into doc in https://github.com/outline/outline/pull/3309
* Empty documents are now cleaned up automatically in https://github.com/outline/outline/pull/3310
* Adds menu item to resend outstanding invites in https://github.com/outline/outline/pull/3348
* Arm64 docker image build by @Yggdrasil80 in https://github.com/outline/outline/pull/3262

#### Editor
* SVGs without a natural px width are no longer invisible in https://github.com/outline/outline/pull/3220
* Editor title does not autoFocus on first load in https://github.com/outline/outline/pull/3238
* Prevent circumstances where history sidebar would crash by @thenanyu in https://github.com/outline/outline/pull/3257
* Catch error when emoji combinations cause document to be unable to persist in https://github.com/outline/outline/pull/3250
* `::` symbols appearing between lines when pasting plaintext in https://github.com/outline/outline/pull/3323
* Improve paste handler parsing for more cases in https://github.com/outline/outline/pull/3322
* Capture file and image drop events below editor in https://github.com/outline/outline/pull/3376
* Added Solidity language support by @fr0zn in https://github.com/outline/outline/pull/3303

#### Platform
* Navigation of shared trees performance in https://github.com/outline/outline/pull/3171
* Hide TOC toggle on publicly shared links if there are no headings in https://github.com/outline/outline/pull/3172
* Don't hide sidebar when menu is open by @iamsaumya in https://github.com/outline/outline/pull/3203
* Don't load CRDT state from database by default in https://github.com/outline/outline/pull/3215
* Stop copying attachments when moving documents in https://github.com/outline/outline/pull/3251
* chore: `documentStructure` database locking in https://github.com/outline/outline/pull/3254
* Add stricter validation around image file type uploads in https://github.com/outline/outline/pull/3324
* Allow admin edit/update access to all collections in https://github.com/outline/outline/pull/3335
* Link preview and search should work on collection descriptions in https://github.com/outline/outline/pull/3355
* Queue retry behavior in https://github.com/outline/outline/pull/3359
* Automatic invite reminder email  in https://github.com/outline/outline/pull/3354
* Lowercase email from auth providers to match any outstanding invites in https://github.com/outline/outline/pull/3369

## New Contributors
* @huruji made their first contribution in https://github.com/outline/outline/pull/3282
* @CommanderRoot made their first contribution in https://github.com/outline/outline/pull/3285
* @fr0zn made their first contribution in https://github.com/outline/outline/pull/3303
* @github-actions made their first contribution in https://github.com/outline/outline/pull/3356
* @Yggdrasil80 made their first contribution in https://github.com/outline/outline/pull/3262

**Full Changelog**: https://github.com/outline/outline/compare/v0.62.0...v0.63.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.63.0'>v0.63.0</a>.</em>

**Comment by bmm-alc** [comment]
kudos to @github-actions :)
